### PR TITLE
Fix/design translation

### PR DESCRIPTION
**Pull request by kjacqueschu**

### CODE REVIEW

File: packages/ui/src/components/filters/CheckboxFilter.tsx [hunk 1]
```diff
@@ -121,9 +118,13 @@ const CheckboxFilter: React.FC<TermFilterProps> = ({
                             type="text"
                         >
                             {isShowingMore
-                                ? get(dictionary, 'actions.less', 'less')
-                                : filteredFilters.length - 5 &&
-                                  `${filteredFilters.length - 5} ${get(dictionary, 'actions.more', 'more')}`}
+                                ? get(dictionary, 'actions.less', 'Less')
+                                : filteredFilters.length - 5 && (
+                                      <>
+                                          <>{`${filteredFilters.length - 5} `}</>
```
Comment (francisl) [inline packages/ui/src/components/filters/CheckboxFilter.tsx:124]: what is the need for child `<>`?

File: packages/ui/src/components/filters/CheckboxFilter.tsx [hunk 2]
```diff
@@ -78,40 +78,37 @@ const CheckboxFilter: React.FC<TermFilterProps> = ({
...
                     </StackLayout>
-                    {filteredFilters
-                        .sort((a, b) => b.data.count - a.data.count)
-                        .slice(0, isShowingMore ? Infinity : maxShowing)
-                        .map((filter, i) => (
-                            <StackLayout
-                                className={styles.checkboxFilterItem}
-                                horizontal
-                                key={`${filterGroup.field}-${filter.id}-${filter.data.count}-${selectedFilters.length}-${i}`}
+                    {filteredFilters.slice(0, isShowingMore ? Infinity : maxShowing).map((filter, i) => (
```
Comment (francisl) [inline packages/ui/src/components/filters/CheckboxFilter.tsx:84]: is it safe to remove the sort for all project?

**Comment by kjacqueschu** [comment]
image
image
image
image

fix hover
image

File: packages/ui/src/components/filters/CheckboxFilter.tsx (hunk 1 above)
Comment (kjacqueschu) [reply packages/ui/src/components/filters/CheckboxFilter.tsx:124]: get fonction can return a React Element so I cant put it inside de `` so to display the return correctly from the get I need to put it inside his own "react element"

File: packages/ui/src/components/filters/CheckboxFilter.tsx (hunk 2 above)
Comment (kjacqueschu) [reply packages/ui/src/components/filters/CheckboxFilter.tsx:84]: ES always sort the data from higher to lower right now. So it should be safe, I put a screenshot why I need to remove it.

We could maybe pass either a sort method down the row or activate it or not. But like I said right now ES does it for us

Comment (francisl) [reply packages/ui/src/components/filters/CheckboxFilter.tsx:84]: 👍

**Review by francisl — APPROVED** [review]
🚢

**Pull request merged**